mavros/src/plugins/setpoint_attitude.cpp

```diff
@@ -26,7 +27,7 @@
 #include <geometry_msgs/PoseStamped.h>
 #include <geometry_msgs/TwistStamped.h>
 #include <mavros_msgs/Thrust.h>
-
+#include <mav_msgs/RollPitchYawrateThrust.h>
```

Comment:
Blank line missed. Do you run uncrustify?

Reply:
In fact I didn't, I will run before the next commit.